File: 프로그래머스/lv1/76501. 음양 더하기/음양 더하기.java
class Solution {
    public int solution(int[] absolutes, boolean[] signs) {
        long answer = 0;
        for(int i = 0; i< absolutes.length; i++) {
            answer += absolutes[i] * (signs[i] ? 1 : -1);
        }
        return (int)answer;
    }
}

File: 프로그래머스/1/76501. 음양 더하기/음양 더하기.go
func solution(absolutes []int, signs []bool) int {
    answer := 0
    for i := 0; i < len(absolutes); i++ {
        if signs[i] {
            answer += absolutes[i]
        }else {
            answer -= absolutes[i]
        }
    }
    return answer
}